iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/NetworkImpl.java

```diff
@@ -1101,13 +1111,15 @@ private void replaceDanglingLineByLine(List<MergedLine> lines, Multimap<Boundary
             l.getTerminal1().setP(mergedLine.p1).setQ(mergedLine.q1);
             l.getTerminal2().setP(mergedLine.p2).setQ(mergedLine.q2);
             mergedLine.properties.forEach((key, val) -> l.setProperty(key.toString(), val.toString()));
+            mergedLine.aliases.forEach(l::addAlias);
```

Comment:
From my point of view, we should automatically add the ids of the half line when a TieLine is created, not only in the merge function. Is there a risk to do it automatically in the add() method of the TieLineAdder?

Reply:
Actually, the TieLineAdder does not have access to the aliases of the half lines, only the IDs. So a dedicated line for merging aliases is still needed.